### PR TITLE
feat(instructions): update instructions

### PR DESCRIPTION
**Pull request by dder725**
### Purpose

Update instructions to reflect current functionality of the app

### Approach

Added new steps with corresponding images, gifs, and text to the Instructions component

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR involve a UI change?

- [ ] Yes
- [x] No

Not a UI change, but content update

### CODE REVIEW

File: src/components/Instructions.tsx [hunk 1]
```diff
@@ -20,67 +28,136 @@ type Step = {
...
     ),
     image: stepOneImg,
   },
   2: {
-    title: "Pick the card",
+    title: "Fill in the details",
     description: (
       <div>
-        You will be presented with the first pair of cards. <br /> If the
+        {/* You will be presented with the first pair of cards. <br /> If the
```
Comment (preetpatel) [inline src/components/Instructions.tsx]: any reason why this commented instruction is here?

Comment (dder725) [reply src/components/Instructions.tsx]: Will remove, forgot about those

**Review by preetpatel — DISMISSED** [review]

**Review by gargi-bansal — CHANGES_REQUESTED** [review]
Unrelated to this but can the arrows on the instructions modal have a pointer cursor? Only if they're enabled though. You could have a not allowed cursor for disabled arrows

**Review by gargi-bansal — DISMISSED** [review]
lgtm!

**Review by gargi-bansal — DISMISSED** [review]

**Review by michellemtan — DISMISSED** [review]
Looks great! Is it possible to change the wording from 'Press' to 'Click' because this is for desktops? Also on my monitor there is some overlap between the instruction page number and some of the instructions i.e. 

image

Otherwise great work!

**Review by mitchfaulcon — CHANGES_REQUESTED** [review]
image
I've got a similar problem as Michelle, but I also have a horizontal scrollbar which I'm not sure should be there - it'd be better if you don't have to scroll to the button

Also when the window is small enough, you can't see the left button

**Review by gargi-bansal — DISMISSED** [review]

File: src/components/Instructions.tsx [hunk 2]
```diff
@@ -20,67 +28,115 @@ type Step = {
...
+      </div>
+    ),
+    image: stepThreeImg,
+  },
+  4: {
+    title: "Create chart and enter the reason of playing",
+    description: (
+      <div>
+        Click Create Chart on your course page. Once you click the button, HeaRT
+        will ask you what is you goal for playing today
```
Comment (mitchfaulcon) [inline src/components/Instructions.tsx]: "will ask you why you are playing today"

File: src/components/Instructions.tsx [hunk 3]
```diff
@@ -20,67 +28,115 @@ type Step = {
...
+        <br />
+        Please tell us why you are playing the game. This helps us understand
+        your needs and continue improving HeaRT tool for you!❤️
```
Comment (mitchfaulcon) [inline src/components/Instructions.tsx]: "your needs and lets us continue improving the HeaRT tool for you"

File: src/components/Instructions.tsx [hunk 4]
```diff
@@ -20,67 +28,115 @@ type Step = {
...
   },
-  4: {
-    title: "Preview and edit your chart",
+  7: {
+    title: "Preview your chart",
     description: (
       <div>
-        Once you went through all teaching dimensions, you can preview and edit
-        your answers before saving your chart 💾
+        Once you went through all teaching dimensions, you can preview your
```
Comment (mitchfaulcon) [inline src/components/Instructions.tsx]: "Once you have been through all teaching dimensions"

File: src/components/Instructions.tsx [hunk 5]
```diff
@@ -20,67 +28,115 @@ type Step = {
...
+        answers and go back to any card you would like to edit. <br />
+        If you are happy with results, don't hesitate to click 'Save Chart'! 💾
```
Comment (mitchfaulcon) [inline src/components/Instructions.tsx]: "With the results"

File: src/components/Instructions.tsx [hunk 6]
```diff
@@ -20,67 +28,115 @@ type Step = {
...
+    title: "Compare charts",
     description: (
       <div>
-        {" "}
-        Save your chart, and compare your answers with your colleagues to see
-        what your perspectives on the course are. Discuss the results, and play
-        the game again as needed! <br />
-        The HeaRT tool is all about sparking the conversation 🔥
+        With HeaRT you can also compare up to three charts to each other in
+        order to see how they align with each other. To do that, select two or
```
Comment (mitchfaulcon) [inline src/components/Instructions.tsx]: "With HeaRT, you can also compare up to three charts to see how they align with each other. To do this,"

Comment (dder725) [reply src/components/Instructions.tsx]: Thanks @mitchfaulcon, awesome job on proofreading! Can I hire you to do the same thing for my P4P report? :D

Comment (mitchfaulcon) [reply src/components/Instructions.tsx]: Mine needs to be written first hahaha

File: src/components/Instructions.tsx [hunk 7]
```diff
@@ -20,67 +28,116 @@ type Step = {
...
+      </div>
+    ),
+    image: stepThreeImg,
+  },
+  4: {
+    title: "Create chart and enter the reason of playing",
+    description: (
+      <div>
+        Click Create Chart on your course page. Once you click the button, HeaRT
+        will ask you why are you playing today
```
Comment (gargi-bansal) [inline src/components/Instructions.tsx]: Full stop?

**Review by mitchfaulcon — DISMISSED** [review]
Thanks for all the changes!

File: src/components/Instructions.tsx [hunk 8]
```diff
@@ -20,67 +28,116 @@ type Step = {
...
+    title: "Choose the position on the dimension bar",
     description: (
       <div>
         When you pick a statement you are presented with a dimension bar. You
         need to select your stance on the dimension: how much does your course
         tend towards one card over the other. <br /> For example, is your course
         more teacher-managed, or is it more student-managed? 🤔 <br />
-        In future versions, you will be able to explain your choice for each
-        dimension in a text note.
+        You can also add a note to explain your choice 📜
```
Comment (gargi-bansal) [inline src/components/Instructions.tsx]: Another one?

**Review by gargi-bansal — DISMISSED** [review]
Left two small comments for missing full stops

**Review by mitchfaulcon — APPROVED** [review]

**Review by gargi-bansal — APPROVED** [review]
lgtm!

**Pull request merged**